Account for the new host_authorization setting introduced in Sinatra
4.1.0.

--- a/lib/capybara/spec/test_app.rb	2025-05-28 08:55:50.434344191 +0200
+++ b/lib/capybara/spec/test_app.rb	2025-05-29 09:00:09.004354200 +0200
@@ -19,6 +19,7 @@
   set :static, true
   set :raise_errors, true
   set :show_exceptions, false
+  set :host_authorization, { permitted_hosts: ['www.example.com', IPAddr.new('0.0.0.0/0')] }
 
   # Also check lib/capybara/spec/views/*.erb for pages not listed here
 
